torch/testing/_internal/faulty_rpc_agent/backend_registry.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+import torch.distributed as dist
+import torch.distributed.rpc as rpc
+import torch.distributed.distributed_c10d as dc10d
+from torch.distributed.rpc import constants as rpc_constants
+
+from caffe2.torch.testing._internal.faulty_rpc_agent import faulty_process_group_agent_pybind
```

Comment:
I assume the `caffe2` prefix would disappear after we move the faulty agent code to `torch/csrc/`?